notifier/service/service.go

```diff
@@ -0,0 +1,223 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/jmoiron/sqlx"
+	"github.com/quay/clair/v4/indexer"
+	"github.com/quay/clair/v4/matcher"
+	"github.com/quay/clair/v4/notifier"
+	namqp "github.com/quay/clair/v4/notifier/amqp"
```

Comment:
Why is this renamed?

Reply:
I've been using samqp for steadway's and namqp for notifier's